localization/english/wc_buildings_l_english.yml

```diff
@@ -269,3 +275,632 @@
 
  building_quilboar_thorns_08:0 "Thorn Labyrinth"
  building_quilboar_thorns_08_desc:1 "This county has been reshaped to a Thorn Labyrinth of Agamaggan, the great boar god who died in battle against the Burning Legion. The Thorn Labyrinth is a massive and intricate network of thorny walls, towers, and tunnels, that cover the entire county. It is a formidable and impenetrable fortress for the quilboar, where they can defend themselves from any invasion or attack."
+ 
+ building_type_gearshaft_university:0 "Gearshaft University"
+ building_type_gearshaft_university_desc:0 "$building_type_generic_university_desc$\n\nThe Gearshaft University is a place for all gnomes to go and learn."
+ building_gearshaft_university:0 "$building_type_gearshaft_university$"
+ building_gearshaft_university_desc:0 "$building_type_gearshaft_university_desc$"
+ building_type_scholomance:0 "Scholomance"
+ building_type_scholomance_desc:0 "$building_type_generic_university_desc$\n\nAlso known as the School of Necromancy, it is a vile academy for prospective necromancers of the Scourge, is located in the ruins of the palatial House of Barov, on a rise overlooking the abandoned city of Caer Darrow."
+ building_scholomance:0 "$building_type_scholomance$"
+ building_scholomance_desc:0 "$building_type_scholomance_desc$"
+ building_type_falthrien_academy:0 "Falthrien Academy"
+ building_type_falthrien_academy_desc:0 "$building_type_generic_university_desc$"
+ building_falthrien_academy:0 "$building_falthrien_academy$"
+ building_falthrien_academy_desc:0 "$building_type_falthrien_academy_desc$"
+ 
+ building_type_deadmines_01:0 "The Deadmines"
+ building_type_deadmines_01_desc:1 "The Deadmines are humanity's greatest source of gold."
+ building_deadmines_01:1 "Deadmining Settlement"
+ building_deadmines_01_desc:1 "$building_type_deadmines_01_desc$"
+ building_deadmines_02:1 "The Deadmines"
+ building_deadmines_02_desc:1 "$building_rammelsberg_mines_02_desc$"
+ building_deadmines_03:1 "Large Deadmines"
+ building_deadmines_03_desc:1 "$building_rammelsberg_mines_03_desc$"
+ building_deadmines_04:1 "Deadmining Complex"
+ building_deadmines_04_desc:1 "$building_rammelsberg_mines_04_desc$"
+ building_type_silvershard_mines_01:0 "Silvershard Mines"
+ building_type_silvershard_mines_01_desc:1 "Diamonds! Stranglethorn Vale's Silvershard mines are full of the gleaming gems, and, while they’re pretty to look at, they’re far more important for what they can buy, new blades, ballistae and battle-gear for the soldiers of whoever owns them."
```

Comment:
```suggestion
 building_type_silvershard_mines_01_desc:1 "Diamonds! Stranglethorn Vale's Silvershard mines are full of the gleaming gems, and while they are pretty to look at, they’re far more important for what they can buy - new blades, ballistae, and battle-gear for the soldiers of whoever owns them."
```

Reply:
I don't see how the original was grammatically incorrect.